### PR TITLE
Fixes for new molecule2

### PR DESCRIPTION
**Pull request by AaronJackson**

### CODE REVIEW

**Comment by manics** [comment]
clamav: Maybe run the verify as `become: root`?

godb: I think it's failing due to a Docker-in-Docker bug:
https://forums.docker.com/t/etc-init-d-docker-62-ulimit-error-setting-limit-invalid-argument-problem/139424
Might be worth downgrading the installed version if there no quick fix?

File: godb/tasks/docker.yml [hunk 1]
```diff
@@ -21,13 +21,34 @@
...
       - containerd.io
+  register: apt_install
+
+- name: temporary fix for docker in docker
+  become: true
+  notify: restart docker
+  ansible.builtin.lineinfile:
+    path: /etc/init.d/docker
+    state: absent
+    regexp: 'ulimit -Hn 524288'
```
Comment (manics) [inline godb/tasks/docker.yml:44]: What's the default systemd ulimit?

Comment (manics) [reply godb/tasks/docker.yml:44]: Ignore that, just realised this is conditional on sysvinit so only for testing 😄. Can you add a comment?

Comment (AaronJackson) [reply godb/tasks/docker.yml:44]: I think it's 524288.
Apparently making it a soft limit would also work (Removing the H flag)
I'm guessing making a smaller value would also work, but we aren't running sysvinit anywhere other than during molecule tests, so I'm not hugely concerned about removing it

**Review by manics — APPROVED** [review]

**Pull request merged**